Vice.js

```diff
@@ -38,12 +38,22 @@
 
 
 // attr()/text() v2 per https://github.com/zotero/translators/issues/1277
-function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+function attr(docOrElem, selector, attr, index) {
```

Comment:
These lines can be deleted completely, as text/attr are now global functions from Zotero itself.

Reply:
I can add a linter rule for that if so desired. Would be an easy rule to do.

Reply:
That would be great! Here is the corresponding issue #1913.

Reply:
https://github.com/zotero/translators/pull/1961